qbft/spectest/generate/state_comparison/tests_MsgProcessingSpecTest/qbft message processing instance proposer.json

```diff
@@ -0,0 +1,76 @@
+{
+		"Share": {
+				"OperatorID": 1,
+				"ValidatorPubKey": "joAGZVGoGzGCWHCe2vfdH2PNaGoOTbiym7t6z+ZWCGd69aUn2USO5Hg1SF4CtQvA",
+				"SharePubKey": "l9lKgR1kSTYFKp0tSs1kcYl0z2eNvv0mcyTI6fjnA0pKa32HeeJ6AZU4w8Qlw+Xn",
+				"Committee": [
+						{
+								"OperatorID": 1,
+								"PubKey": "l9lKgR1kSTYFKp0tSs1kcYl0z2eNvv0mcyTI6fjnA0pKa32HeeJ6AZU4w8Qlw+Xn"
+						},
+						{
+								"OperatorID": 2,
+								"PubKey": "przr4wl9dBcbQMcSoDHOsDcds9PEAs8s5pG5Eg87q3XU1W36DzdZFUSZm/GMU1Pt"
+						},
+						{
+								"OperatorID": 3,
+								"PubKey": "gJDgt2ZqRezF1O90GKyZ8J5sskQCn+pqCn/Mvp7gi8U53g36Zr5rq8hJPdmd0amN"
+						},
+						{
+								"OperatorID": 4,
+								"PubKey": "p8CidrcKXuM5XH1tJlXtYFKKolLU0h7KX8xSI+UMxCvRaLKAq3q1MXNU3d/PPfnk"
+						}
+				],
+				"Quorum": 3,
+				"PartialQuorum": 2,
+				"DomainType": [
+						0,
+						0,
+						3,
+						1
+				],
+				"FeeRecipientAddress": [
+						83,
+						89,
+						83,
+						181,
+						166,
+						4,
+						0,
+						116,
+						148,
+						140,
+						241,
+						133,
+						234,
+						167,
+						210,
+						171,
+						189,
+						102,
+						128,
+						143
+				],
+				"Graffiti": null
+		},
+		"ID": "AQIDBA==",
+		"Round": 1,
+		"Height": 0,
+		"LastPreparedRound": 0,
+		"LastPreparedValue": null,
+		"ProposalAcceptedForCurrentRound": null,
+		"Decided": false,
+		"DecidedValue": null,
+		"ProposeContainer": {
+				"Msgs": {}
```

Comment:
this is empty because we are just ouputting the message and not processing it?

Reply:
Exactly